api/src/main/java/pl/kolendateam/dadcard/items/armor/entity/Armors.java

```diff
@@ -0,0 +1,21 @@
+package pl.kolendateam.dadcard.items.armor.entity;
+
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import pl.kolendateam.dadcard.items.entity.Items;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@DiscriminatorValue(value = "ARMOR")
```

Comment:
podobno jest potrzebna by zidentyfikować poprawną relację miedzy ojcem i dziecmi:

"Discriminator is commonly used in SINGLE_TABLE inheritance because you need a column to identify the type of the record."